_jobs/2019-09-16-test_job copy.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Will that space in the file name screw up the links?

Reply:
(If you can even link to individual job posts)

Reply:
I'd also rename the file from `test_job copy` to something more semantic.

Reply:
Changed the file name thanks!